automated-comments/csharp/general/use_string_interpolation_not_string_concatenation.md

```diff
@@ -0,0 +1 @@
+Rather than using string concatenation, consider using [string interpolation](https://csharp.net-tutorials.com/operators/the-string-interpolation-operator/) to dynamically build a string.
```

Comment:
I would invert this per exercism/automated-mentoring-support#61.

```suggestion
Use [string interpolation](https://csharp.net-tutorials.com/operators/the-string-interpolation-operator/) to dynamically build a string, rather than using string concatenation.
```

Perhaps you want to point out _why_?